src/Tests/Microsoft.NET.Build.Tests/DesignTimeBuildTests.cs

```diff
@@ -81,6 +81,29 @@ public void DesignTimeBuildSucceedsAfterRuntimeIdentifierIsChanged()
             });
         }
 
+        //  Regression test for https://github.com/dotnet/sdk/issues/13513
+        [Fact]
+        public void DesignTimeBuildSucceedsWhenTargetingNetCore21WithRuntimeIdentifier()
+        {
+            var testProject = new TestProject()
+            {
+                Name = "DesignTimePackageDependencies",
+                TargetFrameworks = "netcoreapp2.1",
+                IsSdkProject = true,
+                RuntimeIdentifier = EnvironmentInfo.GetCompatibleRid()
+            };
+
+            testProject.PackageReferences.Add(new TestPackageReference("Newtonsoft.Json", "12.0.2", privateAssets: "All"));
```

Comment:
Are these package references needed? It seems weird to have PrivateAssets on Newtonsoft.Json. And I'm not sure what "Humanizer" does.

Reply:
Indeed, I wonder why these packages are present.  If they're meant to represent some "commonly used dependencies" call that out.